public/components/FeatureAnywhereContextMenu/Container/styles.scss

```diff
@@ -0,0 +1,7 @@
+.context-menu {
+  &__flyout {
+    &.euiFlyout--medium {
+      width: 740px;
```

Comment:
I believe this is copied over from here: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/450#discussion_r1154971204

But pointing out my concerns with that here as well

Reply:
Yes I actually agree, I think we should use small or medium. I missed this while trying to align with Jackie's code. I have the associated detectors style set to medium for the flyout, will check with UX if its okay to go with that for all ad flyouts